tests/e2e/suite_test.go

```diff
@@ -74,6 +74,7 @@ var (
 	netDefaultFileName        string
 	numberOfClusters          int
 	numberOfVMs               int
+	operatorInstallType       string
 	operatorUpgrade           string
```

Comment:
You can just fix the `operatorUpgrade` position and put it after `operatorRepo`. Not related to you change and of course no new VR needed ;-)